File: 프로그래머스/unrated/120813. 짝수는 싫어요/짝수는 싫어요.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int[] solution(int n) {
        List<Integer> tempList = new ArrayList<>();
        for (int i = 1; i <= n; i++) {
            if(i % 2 ==1){
                tempList.add(i);
            }
        }

        int[] answer = new int[tempList.size()];
        for (int i = 0; i < tempList.size(); i++) {
            answer[i] = tempList.get(i);
        }

        return answer;
    }
}

File: 프로그래머스/unrated/120813. 짝수는 싫어요/짝수는 싫어요.js
function solution(n) {
     const tempList = [];
        for (let i = 1; i <= n; i++) {
            if (i % 2 === 1) {
                tempList.push(i);
            }
        }

        const answer = new Array(tempList.length);
        for (let i = 0; i < tempList.length; i++) {
            answer[i] = tempList[i];
        }

        return answer;
}